ui/lib/client/translations/zh.yaml

```diff
@@ -7,6 +7,7 @@ error:
     user:
       signin:
         invalid_code: 授权码无效或已过期
+      lack_required_privileges: 此用户缺少访问 TiDB Dashboard 所需的权限
     slow_query:
       export_no_data: 没有可导出的慢查询日志
     statement:
```

Comment:
Who added this previously! 🤣 It is a wrong place for them.. LOL

Reply:
Let me fix it.